scripts/run_typedoc.sh

```diff
@@ -40,36 +43,67 @@ generate_docs() {
         echo -e "\033[0;93mGenerating typedocs\033[0m"
         pushd ../$2
         if [[ -z "$NOBUILD" ]]; then
+            git clean -xdf
             make ensure && make build && make install
         fi
         if [[ ! -z "$3" ]]; then
             cd $3
         fi
         ${TOOL_TYPEDOC} --json "${PULUMI_DOC_TMP}/$1.docs.json" \
             --mode modules --includeDeclarations --excludeExternals --excludePrivate
+        HEAD_COMMIT=$(git rev-parse HEAD)
         popd
         echo -e "\033[0;93mGenerating pulumi.io API docs\033[0m"
-        ${TOOL_APIDOCGEN} "${PULUMI_DOC_TMP}/$1.docs.json" "${PULUMI_DOC_BASE}/$1"
+        echo -e ${TOOL_APIDOCGEN} "${PULUMI_DOC_TMP}/$1.docs.json" "${PULUMI_DOC_BASE}/$1" $HEAD_COMMIT
+        ${TOOL_APIDOCGEN} "${PULUMI_DOC_TMP}/$1.docs.json" "${PULUMI_DOC_BASE}/$1" $HEAD_COMMIT
     fi
 }
 
-generate_docs "aws" "pulumi-aws" "sdk/nodejs"
-generate_docs "awsx" "pulumi-awsx/nodejs/awsx"
-generate_docs "azure" "pulumi-azure" "sdk/nodejs"
-generate_docs "azure-serverless" "pulumi-azure-serverless/nodejs/azure-serverless"
-generate_docs "cloud" "pulumi-cloud/api"
-generate_docs "cloud-aws" "pulumi-cloud/aws"
-generate_docs "cloud-azure" "pulumi-cloud/azure"
-generate_docs "cloudflare" "pulumi-cloudflare" "sdk/nodejs"
-generate_docs "docker" "pulumi-docker" "sdk/nodejs"
-generate_docs "eks" "pulumi-eks/nodejs/eks"
-generate_docs "f5bigip" "pulumi-f5bigip" "sdk/nodejs"
-generate_docs "gcp" "pulumi-gcp" "sdk/nodejs"
-generate_docs "kubernetes" "pulumi-kubernetes" "sdk/nodejs"
-generate_docs "openstack" "pulumi-openstack" "sdk/nodejs"
-generate_docs "packet" "pulumi-packet" "sdk/nodejs"
-generate_docs "pulumi" "pulumi/sdk/nodejs"
-generate_docs "random" "pulumi-random" "sdk/nodejs"
-generate_docs "vsphere" "pulumi-vsphere" "sdk/nodejs"
+REPOS=(
+    "aws,pulumi-aws,sdk/nodejs"
+    "awsx,pulumi-awsx/nodejs/awsx"
+    "azure,pulumi-azure,sdk/nodejs"
+    "azure-serverless,pulumi-azure-serverless/nodejs/azure-serverless"
+    "cloud,pulumi-cloud/api"
+    "cloud-aws,pulumi-cloud/aws"
+    "cloud-azure,pulumi-cloud/azure"
```

Comment:
@lukehoban i think you're removin azure-serverless/cloud-aws/cloud-azure.  Do you just want me to do it?